Sources/XiCliCore/CommandLineTool.swift

```diff
@@ -52,6 +52,16 @@ public final class CommandLineTool {
     func resolvePath(from input: String) throws -> String {
         let fileManager = FileManager.default
         var filePath: URL!
+
+        // Small helper function used to determine if path is not a folder.
+        func pathIsNotDirectory(_ path: String) -> Bool {
+            var isDirectory = ObjCBool(false)
+            if fileManager.fileExists(atPath: path, isDirectory: &isDirectory) {
+                return !isDirectory.boolValue
+            } else {
+                return false
```

Comment:
I think we can now return `true` here :)

Reply:
You're right, going to fix that :)